src/Symfony/Component/Validator/Constraints/Callback.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
+use Attribute;
```

Comment:
we generally don't use use statements for classes of the global namespace. Is this one necessary ?

Reply:
Indeed, good catch.

Reply:
It isn't. If I removed the `use`, I would have to fully qualify all references to the attribute class. In this specific case, this line
```php
#[Attribute(Attribute::TARGET_CLASS | Attribute::TARGET_PROPERTY | Attribute::TARGET_METHOD | Attribute::IS_REPEATABLE)]
```
would look like this.
```php
#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
```

Apart from increased ugliness, there's no disadvantage. 😃

Importing `Attribute` might be my personal preference, but if you prefer to have those backslashes, I can change this. And I would also submit a PR that adjust the attributes from my previous two PRs.

Reply:
> adjust the attributes from my previous two PRs.

that'd be consistent with the codebase yes.
Please do so in this very PR to reduce the process-overhead.

Reply:
All right.

Reply:
All `use Attribute;` statements are gone now.